diff --git a/CMakeLists.txt b/CMakeLists.txt
index 369944c..630f686 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -22,8 +22,8 @@ set(CMAKE_CXX_STANDARD_REQUIRED True)
 # - Intel (not regularly tested)
 # other compilers on a best effort
 
-add_subdirectory(Test/ClangTest)
-add_subdirectory(Test/GccTest)
-add_subdirectory(Test/MsvcTest)
+# add_subdirectory(Test/ClangTest)
+# add_subdirectory(Test/GccTest)
+# add_subdirectory(Test/MsvcTest)
 
-enable_testing()
\ No newline at end of file
+# enable_testing()
\ No newline at end of file
